custom_components/acwd/acwd_api.py

```
@@ -70,20 +97,24 @@ def login(self):
-                'Content-Type': 'application/json; charset=UTF-8',
-                'Referer': self.base_url,
-                'X-Requested-With': 'XMLHttpRequest',
-                'User-Agent': USER_AGENT,
-                'CSRFToken': csrf_token
-            }
-        )
-
-        if update_state_response.status_code != 200:
-            logger.warning(f"updateState returned {update_state_response.status_code}")
+        try:
+            update_state_response = self._retry_request(
```

Ruff (0.14.8)

Local variable `update_state_response` is assigned to but never used

Remove assignment to unused variable `update_state_response`

(F841)